Fix NPE on SkullBuilder

Lorenzo0111 opened the pull request.

NichtStudioCode commented:
Hi, thanks for notifying me about the issue. I fixed it in a5ad22cc2badfa19fe189d26359f0cca34148a25.
The CodeMC repository is unreliable and will not be added to InvUI.

Lorenzo0111 commented:
> Hi, thanks for notifying me about the issue. I fixed it in [a5ad22c](https://github.com/NichtStudioCode/InvUI/commit/a5ad22cc2badfa19fe189d26359f0cca34148a25).
> The CodeMC repository is unreliable and will not be added to InvUI.
 
 thanks, I just added that repository to make it compile on JitPack so I was able to use the lib without waiting for you
